src/bpffeature.cpp

```diff
@@ -0,0 +1,97 @@
+#include <bcc/libbpf.h>
+#include <bpffeature.h>
+#include <fcntl.h>
+#include <stddef.h>
+#include <stdio.h>
+#include <unistd.h>
+
+#include "utils.h"
+
+namespace libbpf {
+#undef __BPF_FUNC_MAPPER
+#include "libbpf/bpf.h"
+} // namespace libbpf
+
+namespace bpftrace {
+
+static bool try_load(const char* name,
+                     bpf_prog_type prog_type,
+                     struct bpf_insn* insns,
+                     size_t len)
+{
+  int log_size = 40960;
```

Comment:
@fbs 
When I tried to compile with clang++ I got the following error.

```sh
/home/ubuntu/bpftrace/src/bpffeature.cpp:23:15: error: variable-sized object may not be initialized
  char logbuf[log_size] = {};
              ^~~~~~~~
```

Using `constexpr` fixed this.

```suggestion
  constexpr int log_size = 40960;
```

Reply:
Annoying. Never tested building with llvm.  can you submit a patch?

Reply:
the android build I have been working on uses clang, so this would hit me too.

I also had an experimental flag to force using clang as the toolchain, as it resulted in about 5MB smaller binaries.

If someone else doesn't fix this I probably will hit it and fix it soon.

Reply:
Just created #1092. I also found clang generated several warnings.

Reply:
maybe we should add an clang build to the CI? Would be good to catch issues like this

Reply:
@fbs that was my plan already actually! The Android PR (still rebasing it on #1041 before I submit it) will do this implicitly. I also have a much smaller one that just swaps out the toolchain for Claing. I can get both of these up shortly, the main thing blocking them is getting Github Actions enabled, which @danobi is working on.